### ISSUE TITLE
Feature proposal: (dark) background

### ISSUE THREAD

**Issue by ihatemicrosoft-was-already-taken** [HenriWahl/Nagstamon#591]
I would like to be able to change the background of the window or fullscreen display to dark grey/black instead of white. Would be nice if the other UI elements could then change (invert?) colors as well.

**Comment by HenriWahl** [comment]
macOS has dark mode now, Linux GNOME at least too - what OS do you use?

**Comment by aw-think** [comment]
What about a dark mode for Windows 10 (x64/x86)? A dark mode would be nice...

**Comment by ihatemicrosoft-was-already-taken** [comment]
> macOS has dark mode now, Linux GNOME at least too - what OS do you use?

Windows 8.1 and Windows 10

**Comment by HenriWahl** [comment]
As far as I found out Qt5 does not have a special darkmode for Windows.

**Comment by aw-think** [comment]
Thats true, but this might be an example: https://forum.qt.io/topic/101391/windows-10-dark-theme/4

**Comment by HenriWahl** [comment]
Latest testing versions of Nagstamon use Qt6 - does this help?
